include/boost/geometry/algorithms/detail/equals/collect_vectors.hpp

```diff
@@ -62,11 +62,11 @@ struct collected_vector
     : nyi::not_implemented_tag
 {};
 
-// compatible with side_by_triangle cartesian strategy
+// compatible with side_robust cartesian strategy
 template <typename T, typename Geometry, typename CT, typename CSTag>
 struct collected_vector
     <
-        T, Geometry, strategy::side::side_by_triangle<CT>, CSTag
+        T, Geometry, strategy::side::side_robust<CT>, CSTag
```

Comment:
All cartesian strategies should probably be suppoted `collected_vector`.

Reply:
I created a specialization for `side_by_triangle` for backwards compatibility.